spec/Cabin/Infotainment.vspec

```diff
@@ -49,6 +49,12 @@ Media.Played.URI:
   type: sensor
   description: User Resource associated with the media
 
+Media.Played.PlaybackSpeed:
+  datatype: float
+  type: actuator
+  allowed: [0.25, 0.5, 0.75, 1, 1.25, 1.5, 1.75, 2]
```

Comment:
Is the restrictions to only allow those particular values common, or even stated in a specification somewhere? I noticed that e.g. Android also use a float, but does not state any limitations, see eg., https://developer.android.com/reference/android/media/PlaybackParams#setSpeed(float)

Reply:
@erikbosch Thanks a lot for quick review! I agreed to remove the restrictions of PlaybackSpeed. I think the restriction probably can be covered by applications side. So I will remove this.

Reply:
If needed we could find a way as well how to describe these rates in allowed tag. But as well current proposal should be good enough.